synapse/lib/stormlib/env.py

```diff
@@ -21,10 +20,10 @@ class LibEnv(s_stormtypes.Lib):
             'type': 'function', '_funcname': '_libEnvGet',
             'args': (
                 {'name': 'name', 'type': 'str', 'desc': 'The name of the environment variable.', },
-                {'name': 'default', 'type': 'obj', 'default': None,
+                {'name': 'default', 'type': 'prim', 'default': None,
                     'desc': 'The value to return if the environment variable is not set.', },
             ),
-         'returns': {'type': 'str', 'desc': 'The environment variable string.'},
+            'returns': {'type': 'str', 'desc': 'The environment variable string.'},
```

Comment:
Should this be `str, prim` since default accepts a prim?

Reply:
It would probably be better as `['str', 'null']` actually ( to account for the default value for `default` ). Someone passing in a non-str into that default value is really just creating their own type-confusion headache; or we should `tostr(default, noneok=True)` there.